alerts/plugins/ip_source_enrichment.json

```diff
@@ -0,0 +1,14 @@
+{
+  "known": [
+    {
+      "range": "8.32.0.0/16",
+      "site": "OFFICE1",
+      "format": "{0} is in OFFICE1."
```

Comment:
something like "format": "{0} is in {1}." where {1} is "site" value?

Reply:
I don't see the benefit of including a formatted string, as long as the data we care about is stored in "site"?

Reply:
I think there are bits of info we'd like to include that might not make sense to put in 'site' or somewhere else.  It might just be some really specific description text